packages/main/src/FileUploader.js

```diff
@@ -284,6 +285,12 @@ class FileUploader extends UI5Element {
 		});
 	}
 
+	_onkeydown(event) {
```

Comment:
Usually, for all components, Enter is handled on keydown (as here),
and Space on keyup - here it is handled on keydown. Could you check this?